### ISSUE TITLE
Don't get a chance to read XP/gold result if a drop occurs

### ISSUE THREAD

**Issue by SabreCat** [HabitRPG/habitica#1045]
If a potion or egg drops when I accomplish something, the screen goes dark except the popup, all as intended. However, by the time I acknowledge the drop popup, the overlay showing my XP and gold gain from the click is gone. Would be nice to have that wait until after acknowledging the drop.

**Comment by SabreCat** [comment]
Also an issue with streak achievement notifications.

**Comment by paglias** [comment]
Hi! The site has been totally rewritten from the ground up, I'm closing the issue since it's probably fixed. Check if it's still happening and in case reopen the bug, thanks! Notifications are still missing!

**Comment by SabreCat** [comment]
This is only a non-issue right now because we don't have popup drop notifications in the rewrite yet. It's something we can keep an eye out for when that's re-implemented, though, so I'm reopening.

**Comment by paglias** [comment]
Drops notification are back in but I think this still occurs, right^

**Comment by paglias** [comment]
We could

a) Delay notifications until modal is closed (see http://stackoverflow.com/questions/11952579/watch-multiple-scope-attributes-in-angularjs)
b) Use some css trick to show both together

**Comment by paglias** [comment]
Would go with **a** but the problen is that when a drop occurred is known to angular only after the server returns data (right, @lefnire?) while the gp/exp points are updated client side so they're not concurrent

**Comment by lefnire** [comment]
that's correct. I'd go with b personally, just give the growl updates a really high z-index. Currently they're hidden under username dropdown anyway, so that would solve two birds one stone

**Comment by paglias** [comment]
yes **b** is working, pushing now

**Comment by lefnire** [comment]
Awesome!
